src/components/button/button.svelte

```diff
@@ -108,6 +108,18 @@
   :host button {
     width: 100%;
   }
+
+  .leoButton {
+    --foreground: var(--leo-color-text-primary);
+    --background: var(--leo-color-container-background);
```

Comment:
WE were just discussing how these variable names are pretty finite in that they seem like they will be used for `foreground` and `background` on the button, but they are simply the foreground and background "base" colors that will be mixed with other colors. Perhaps at least just a comment here saying that's what they are, as I can't really think of a better variable name - they are understandable after all.